### DISCUSSION TITLE
VSCode shows error when adding plugin-graphql to Nuxt's nuxt.config.ts

### DISCUSSION THREAD

**Discussion by sdr0x07b6** [rollup/plugins#1592] [Q&A]
nuxt.config.ts
```ts

import graphql from '@rollup/plugin-graphql'

export default defineNuxtConfig({
  ...

  vite: {
    // for Frontend
    plugins: [graphql()],
  },

  nitro: {
    rollupConfig: {
      // for Backend
      plugins: [graphql()],
    },
  },
});
```

When added in this way, it works fine in operation.
However, on the VSCode editor, an error is displayed at the `graphql()` point on the nitro side.
```
Type instantiation can be very deep and infinite. ts(2589)
(alias) graphql(options?: RollupGraphqlOptions | undefined): Plugin<any>
import graphql
Convert .gql/.graphql files to ES6 modules
```

Is this an error that only occurs with nuxt.config.ts?
How can I resolve this?

- @rollup/plugin-graphql v2.0.3
- Nuxt v3.7.3
- Nodejs v18.18.0
- macOS 13.5 Apple Silicon M2

**Comment by shellscape** [comment]
Converted to Q&A discussion as this doesn't meet our requirements for an issue. Might want to run this by vscode or nuxt folks

**Comment by paulvonber** [comment]
exactly same issue, anybody got clue how to fix it?
